ms-ml/app/routers/train.py

```
@@ -172,6 +172,253 @@ async def train_wait_time(request: WaitTimeTrainRequest):
+    cutoff_30d = now - pd.Timedelta(days=30)
+
+    for _, row in df.iterrows():
+        book_id = row["bookId"]
+
+        # historical_avg_wait_days: promedio de reservas fulfilled ANTES de esta
+        hist_pipeline = [
+            {
+                "$match": {
+                    "bookId": row["bookId"]
+                    if isinstance(row["bookId"], str)
+                    else row["bookId"],
```

Ruff (0.15.20)

[warning] Useless `if`-`else` condition

(RUF034)